plenum/server/validator_info_tool.py

```diff
@@ -165,8 +195,258 @@ def __unreachable_list(self):
+        indy_packages = ret.stdout.split(os.linesep)
+
+        return {
+            "Software": {
+                "OS_version": os_version,
+                "Installed_packages": installed_packages,
+                # TODO add this field
+                "Indy_packages": self._prepare_for_json(indy_packages),
+            }
+        }
+
+    def _cat_file(self, path_to_file):
+        ret = subprocess.run("cat {}".format(path_to_file),
+                             shell=True,
+                             check=True,
+                             universal_newlines=True,
+                             stdout=subprocess.PIPE,
+                             timeout=5)
+        return ret.stdout.split(os.linesep) if ret.returncode == 0 else []
+
+    def _get_genesis_txns(self):
+        genesis_txns = {}
+        genesis_pool_txns_path = os.path.join(genesis_txn_path(self._node.genesis_dir,
+                                                               self._node.config.poolTransactionsFile))
+        genesis_domain_txns_path = os.path.join(genesis_txn_path(self._node.genesis_dir,
+                                                                 self._node.config.domainTransactionsFile))
+        genesis_config_txns_path = os.path.join(genesis_txn_path(self._node.genesis_dir,
+                                                                 self._node.config.configTransactionsFile))
+        if os.path.exists(genesis_pool_txns_path):
+            genesis_txns['pool_txns'] = self._cat_file(genesis_pool_txns_path)
+        if os.path.exists(genesis_domain_txns_path):
+            genesis_txns['domain_txns'] = self._cat_file(genesis_domain_txns_path)
+        if os.path.exists(genesis_config_txns_path):
+            genesis_txns['config_txns'] = self._cat_file(genesis_config_txns_path)
+
+        return genesis_txns
+
+    def _get_configs(self):
+        main_config = []
+        network_config = []
+        user_config = []
+        path_to_main_conf = os.path.join(self._node.config.GENERAL_CONFIG_DIR,
+                                         self._node.config.GENERAL_CONFIG_FILE)
+        if os.path.exists(path_to_main_conf):
+            main_config = self._cat_file(path_to_main_conf)
+
+        network_config_dir = os.path.join(self._node.config.GENERAL_CONFIG_DIR,
+                                          self._node.config.NETWORK_NAME)
+        if os.path.exists(network_config_dir):
+            path_to_network_conf = os.path.join(network_config_dir,
+                                                self._node.config.NETWORK_CONFIG_FILE)
+            network_config = self._cat_file(path_to_network_conf)
+        if self._node.config.USER_CONFIG_DIR:
+            path_to_user_conf = os.path.join(self._node.config.USER_CONFIG_DIR,
+                                             self._node.config.USER_CONFIG_FILE)
+            user_config = self._cat_file(path_to_user_conf)
+
+        return {"Main_config": main_config,
```

Comment:
I would prefer to not return configs in this json as they can also be jsons, and may be quite big.